student-store-express-api/server.js

```diff
@@ -3,5 +3,5 @@ const app = require("./app")
 const port = process.env.PORT || 3001
 
 app.listen(port, () => {
-  console.log(`🚀 Server listening on port ` + port)
+  // console.log(`🚀 Server listening on port ` + port)
```

Comment:
This is a helpful line to keep uncommented out since it can show that your server is active